lib/quby/questionnaires/entities/questionnaire.rb

```diff
@@ -183,9 +181,20 @@ def to_codebook(options = {})
             output << ""
           end
 
+          flags.each do |_flag_key, flag|
```

Comment:
Nou het is ook een snelheidsding, het voelt gewoon vies om een boel `*blah.map{}.flatten` in een grote array te zetten. De << (streaming/appending) aanpak is de correcte, ik zal het nog herschrijven.